src/traffic_top/stats.h

```diff
@@ -63,7 +61,7 @@ const char end[]       = "\",\n";
 class Stats
 {
 public:
-  Stats(const string &url) : _url(url)
+  Stats(const std::string &url) : _url(url)
```

Comment:
It looks nice idea to reduce changes. Will do.

Reply:
Done. Actually, type alias couldn't be used with `std::make_pair`, because it's template function.
I add a simple private function instead of general template function alias like below.
Because all of `std::make_pair` in this class take the same types. 

```
template <typename... Args>
auto
make_pair(Args &&... args) -> decltype(std::make_pair(std::forward<Args>(args)...))
{
  return std::make_pair(std::forward<Args>(args)...);
}
```